### ISSUE TITLE
Resource android:attr/lStar not found error while building the app on Android

### ISSUE THREAD

**Issue by PMFlis** [FlutterFlow/flutterflow-issues#3942]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

While building the app on Android I get this error: ERROR:/Users/paulina/Desktop/storme/build/assets_audio_player_web/intermediates/merged_res/release/values/values.xml:194: AAPT: error: resource android:attr/lStar not found.
I use in my app the flutter flow AudioPlayer widget that seems to be causing the problem. 
I have created completely new test app that contains only the AudioPlayer widget and the error persists:  ERROR:/Users/paulina/Desktop/test/build/assets_audio_player_web/intermediates/merged_res/release/values/values.xml:194: AAPT: error: resource android:attr/lStar not found.
The app works on both the ios and android simulator, builds on ios but it doesn't build on android due to the error described above. 

### Expected Behavior

Build the app on Android without the error and as well be able to use the flutterflow widget: AudioPlayers

### Steps to Reproduce

1. Download the code
2. Open the code in VSCode
3. Run command: flutter build apk
4. The error appears 

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITESz/Hqx89gocgF7s/xY/pQhQUkKVwkaIJEs+1uZ00jF5D4POwTPcmkaRNvYMSzaH5iO1CKkmMJw8LuuNj9E/Q6MgeAfN1YzZZTFxPPbGGiMqqhPbqOOnNCO8BUG3aG0ImCkCRCGNNtWmAA3GaIFdm+RT2Cf9qOYwx5e6fDbOY=

### Visual documentation

![Screenshot 2024-09-30 at 16 53 07](https://github.com/user-attachments/assets/14803b30-51ea-477e-916f-5f18a0268c8d)
<img width="1470" alt="Screenshot 2024-09-30 at 17 00 42" src="https://github.com/user-attachments/assets/fc0adc9f-2a72-4891-bfb1-b894e449943e">

### Environment

```markdown
- FlutterFlow version: v5.0.1 
- Platform: Android
- Browser name and version: Desktop app
- Operating system and version affected: Mac
```

**Comment by Alezanello** [comment]
Hello!

I downloaded the code from the test project you provided and successfully ran the "flutter build apk" without any issues. Could you please run `flutter doctor -v` on your local machine and share the log? This will help us confirm if you're using the correct Flutter version locally.

Looking forward to your response!

Best regards,<br>Azanello

<img src="https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/74936659-d7cd-478d-ae59-c4daa0c5e0a6/ce9a735d-5d88-4c58-9b9c-4e5540c10902?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS83NDkzNjY1OS1kN2NkLTQ3OGQtYWU1OS1jNGRhYTBjNWUwYTYvY2U5YTczNWQtNWQ4OC00YzU4LTliOWMtNGU1NTQwYzEwOTAyIiwiaWF0IjoxNzI3ODE1NTY2LCJleHAiOjMzMjk4Mzc1NTY2fQ.ZZ59d11Hq-hr9hdldYioDu2ebmi9H38De9-IM0B7knM " alt="image.png" width="1864" height="921" />

<img src="https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/720f6997-a13e-4fd0-806a-a78beed9de9b/66ce450a-a5e2-4196-91a1-dc21d5073b32?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS83MjBmNjk5Ny1hMTNlLTRmZDAtODA2YS1hNzhiZWVkOWRlOWIvNjZjZTQ1MGEtYTVlMi00MTk2LTkxYTEtZGMyMWQ1MDczYjMyIiwiaWF0IjoxNzI3ODE1NTY2LCJleHAiOjMzMjk4Mzc1NTY2fQ.TAL6qQU3H0gp7MJsdShuPt1z4OEPJ4t1yYMKNyb0Vaw " alt="image.png" width="905" height="554" />

**Comment by PMFlis** [comment]
Hi @Alezanello, I have attached the screenshots after running flutter doctor -v. I have asked my colleague to download the code from my ff account and build it on his mac and he got exactly same error as me. 
<img width="1470" alt="Screenshot 2024-10-02 at 13 27 18" src="https://github.com/user-attachments/assets/e3d1df04-7c07-4046-bcbb-3421950f08e0">
![flutterflow_bug_flutter_doctor](https://github.com/user-attachments/assets/a4500065-3b9f-4f12-80e8-73671888698f)

**Comment by Alezanello** [comment]
Thank you for sharing the logs! I noticed that you're using Flutter version 3.24.3, while FlutterFlow currently supports version 3.22.2, as shown in the logs I shared. To ensure everything compiles correctly, I recommend downgrading your Flutter version to 3.22.2.

<img src="https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/036ec5c6-cc6c-4487-aef0-dc600a081b73/75c0b42f-90aa-4161-bac0-88dec7ee33e3?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS8wMzZlYzVjNi1jYzZjLTQ0ODctYWVmMC1kYzYwMGEwODFiNzMvNzVjMGI0MmYtOTBhYS00MTYxLWJhYzAtODhkZWM3ZWUzM2UzIiwiaWF0IjoxNzI3ODcyNTgxLCJleHAiOjMzMjk4NDMyNTgxfQ.7nXShZaFz1rNYVHPas0BSbvNf7QH2o8uKAEYFHZkK90 " alt="image.png" width="306" height="166" />
